src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentOneDay = 40;
+  const discountMoreFiveDays = 50;
+  const discountMoreThreeDays = 20;
```

Comment:
Consider replacing the hardcoded literals `40`, `50`, and `20` with descriptive constants. This will make the code more readable and maintainable. For example, you could define constants like `DAILY_RENTAL_COST`, `DISCOUNT_FOR_SEVEN_DAYS`, and `DISCOUNT_FOR_THREE_DAYS`.